include/psa/crypto_config.h

```diff
@@ -88,6 +88,8 @@
 #define PSA_WANT_ALG_STREAM_CIPHER              1
 #define PSA_WANT_ALG_TLS12_PRF                  1
 #define PSA_WANT_ALG_TLS12_PSK_TO_MS            1
+#define PSA_WANT_ALG_TLS12_ECJPAKE_TO_PMS       1
+
```

Comment:
```suggestion
```

Reply:
This was deliberately added to separate two sets of defines, but if you feel that it's out of place - I can remove it :)